Resources/Prototypes/Entities/Effects/ambient_sounds.yml

```
@@ -0,0 +1,10 @@
+- type: entity
+  id: AmbientSoundSourceFlies
+  name: ambient sound source
+  suffix: flies
+  components:
+  - type: AmbientSound
+    volume: -5
+    range: 3
+    sound:
+      path: /Audio/Ambience/Temporary/flies.ogg
```

```suggestion
  - type: AmbientSound
    volume: -5
    range: 3
    sound:
      path: /Audio/Ambience/Temporary/flies.ogg
  - type: Tag
    tags:
    - HideContextMenu
```

shouldn't happen normally but best to do this for all the abstract-type entities like this